HISTORY.md

```diff
@@ -0,0 +1,47 @@
+## 2.0.0 (2020-02-13)
```

Comment:
does the format work for the changelog generator?

Reply:
Yes, it should be exactly what it is after.  As well as moving the content, I fixed up the headings, added dates etc.